action.yml

```
@@ -0,0 +1,37 @@
+name: "Set up Wasp App Runner"
+description: |
+  Set up Wasp App Runner for use in GitHub Actions workflows
+
+  **Note:** You must set up Node.js before using this action.
+
+inputs:
+  path:
+    description: "Path where to checkout the Wasp App Runner repository"
+    required: false
+    default: ".wasp-app-runner"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout wasp-app-runner
+      uses: actions/checkout@v4
+      with:
+        repository: wasp-lang/runner-action
+        path: ${{ inputs.path }}
+
+    - name: Cache npm cache
+      uses: actions/cache@v4
+      with:
+        path: ${{ inputs.path }}/.npm-cache
+        key: npm-wasp-app-runner-${{ runner.os }}-${{ hashFiles(format('{0}/package-lock.json', inputs.path)) }}
+        restore-keys: |
+          npm-wasp-app-runner-${{ runner.os }}-
+
+    - name: Install dependencies and Wasp App Runner
+      shell: bash
+      working-directory: ${{ inputs.path }}
+      env:
+        npm_config_cache: ${{ github.workspace }}/${{ inputs.path }}/.npm-cache
+      run: |
+        npm ci
+        npm run install:global
```

Personal taste: I'd split this into two lines
```suggestion
        npm run build
        npm install --global .
```
And remove the `install:global` script.
Mainly because I don't like a package having a script that affects global environment (this is one of my gripes with `waspc/run` as well).

Not blocking, I could be convinced of the opposite opinion so deferring to you, it's fine if you keep it like that.

---

Hm. For me this is closely related to `prepare`.
If we have `prepare` then global install is trivial, and we don't really need this.

If we remove `prepare` then we need to know it. `install:global` was a way to avoid this. But once again we need to know that `install:global` exists, bad again.

I don't want to know the build process when trying to install this package.

That's why I'm maybe leaning into:
1) `leave` prepare
2) remove `install:global`

-> you use native `npm` stuff to install and it just works?